dotcom-rendering/src/web/components/PinnedPost.tsx

```diff
@@ -154,23 +155,16 @@ export const PinnedPost = ({ pinnedPost, children, format }: Props) => {
 				name="pinned-post-checkbox"
 				tabIndex={-1}
 				key="PinnedPostCheckbox"
+				aria-controls="collapsible-body"
+				aria-label="Toggle pinned post"
 			/>
-			<div css={rowStyles(palette)}>
-				<SvgPinned />
-				{pinnedPost.blockFirstPublished && (
-					<time data-relativeformat="med" css={timeAgoStyles}>
-						From {timeAgo(pinnedPost.blockFirstPublished)}
-					</time>
-				)}
-			</div>
-			<div id="collapsible-body" css={collapsibleBody}>
-				{children}
-			</div>
-			<div id="pinned-post-overlay" css={overlayStyles} />
 			<label
 				css={fakeButtonStyles(palette)}
 				htmlFor="pinned-post-checkbox"
 				id="pinned-post-button"
+				// eslint-disable-next-line jsx-a11y/no-noninteractive-tabindex
```

Comment:
Can you explain how this is an interactive `tabindex`? Or why the rule is required?

```ts
// eslint-disable-next-line rule-to-disable -- reason for disabling the rule
```